src/Scene/NodeProcess.js

```diff
@@ -84,6 +84,36 @@ define('Scene/NodeProcess',
 
     };
 
+    NodeProcess.prototype.subdivide = function (node, camera, params) {
```

Comment:
This function only uses `params.tree` so it would make sense to change the third argument to `quadtree`